agent_payload.proto

```diff
@@ -75,36 +75,10 @@ message SketchPayload {
 			repeated uint32 delta = 9;
 			repeated double buf = 10;
 		}
-		message DistributionK {
-			message Compactor {
-				repeated double v = 1;
-			}
-			int64 ts = 1;
-			int64 cnt = 2;
-			double min = 3;
-			double max = 4;
-			double avg = 5;
-			double sum = 6;
-			repeated Compactor compactors = 7 [(gogoproto.nullable) = false];
-			uint32 length = 8;
-			uint32 capacity = 9;
-			uint32 h = 10;
-		}
-		message DistributionC {
-			int64 ts = 1;
-			int64 cnt = 2;
-			double min = 3;
-			double max = 4;
-			double avg = 5;
-			double sum = 6;
-			repeated double vals = 7;
-		}
 		string metric = 1;
 		string host = 2;
 		repeated Distribution distributions = 3 [(gogoproto.nullable) = false];
 		repeated string tags = 4;
-		repeated DistributionK distributionsK = 5 [(gogoproto.nullable) = false];
```

Comment:
had a look at the protobuf docs and how to remove fields in a backward- and forward-compatible way: apparently here we should use the `reserved` keyword to make sure we never re-use these tag numbers again in future fields: see https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
Thanks for the pointer - fixed!